### DISCUSSION TITLE
Where is the v8.4 of ImageGlass ?

### DISCUSSION THREAD

**Discussion by Zeus64** [d2phap/ImageGlass#1539] [General]
Where can I find the version 8.4 of ImageGlass ? How to check if ImageGlass v8.4 was one day installed in my computer ?

**Comment by d2phap** [comment]
ImageGlass 8.4 was deleted. You can see full list of your installed app by right-clicking on Start button, select "Installed apps" (for Windows 11), or "Apps and features" (for Windows 10)
